app/controllers/AdminController.scala

```diff
@@ -669,4 +669,139 @@ class AdminController @Inject() (implicit val env: Environment[User, SessionAuth
       }
     )
   }
+
+  /**
+   * Gets street edge data for the coverage section of the admin page.
+   */
+  def getStreetEdgeData = UserAwareAction.async { implicit request =>
+    if (isAdmin(request.identity)) {
```

Comment:
Let's actually get rid of the admin requirement for this. We have it in a few other places on the admin page where I don't think it's necessary, we had just been doing it by default because the data was for the admin page. But let's make it more open unless there's a reason not to!